projects/admin/src/app/record/detail-view/document-detail-view/files-collections/files-collections.component.ts

```diff
@@ -69,6 +72,17 @@ export class FilesCollectionsComponent implements OnInit, OnDestroy {
     }
   }
 
+  /**
+   * Generate the public interface collection search link.
+   *
+   * @param name - the collection name
+   * @returns - url on the public interface
+   */
+  getCollectionLink(name): string {
+    const viewcode = this.organisationService.organisation.code;
+    return `https://localhost:5000/${viewcode}/search/documents?q=files.collections.raw:${name}&simple=0`;
```

Comment:
Why the URL is `https://localhost:5000` ?
Maybe you need to change that.

Reply:
It was for debug. Thanks for the reporting.